### DISCUSSION TITLE
Docker log file very large

### DISCUSSION THREAD

**Discussion by roblomq** [zwave-js/zwave-js-ui#2832] [Q&A]
Today I found out the the logfile of the docker container where zwave-js-ui is running is was quite large 6GB+
I'm quite a noob with this but I've managed to delete the log files. But I assume these logfiles will be recreated and grow again. Is there something I can do to stop this (by logrotate and or limit the log file size)?

After googling I found someone who said that the following code needs to be added in the yaml file
`    logging:
      driver: "json-file"
      options:
        max-size: "10m"
        max-file: "10"
`
But do I then have to recreate the container and if yes then I assume all data in it (and therefore zwave-js-ui) is gone. SO hope there is another way?

**Comment by robertsLando** [comment]
> But do I then have to recreate the container and if yes then I assume all data in it (and therefore zwave-js-ui) is gone

Nope data in container will not be lost by changing the logging driver, simply update your docker-compose file and run `docker-compose up -d`. I also suggest to enable scheduled backups in settings (of both store and nvm)

**Comment by roblomq** [comment]
Good to know that you can update a container without destroying the data. 
As my knowledge on this is nearly zero, can you suggest a how to or example about the scheduled backup?

**Reply by robertsLando** [reply]
> Good to know that you can update a container without destroying the data.

The important thing is to have a volume, the sensitive data are stored there

About the scheduled backup just go to settings and enable it, there are both store and nvm backups, the first backups your settings and preferences the other one backups the stick.
